apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/AplCoreRuntime.java

```diff
@@ -30,6 +31,8 @@
  * @author alukin@gmail.com
  */
 public class AplCoreRuntime {
+    //probably it is temprary solution, we should move WebUI serving out of core 
+    public final static String WEB_UI_DIR="webui";
     private static Logger LOG = LoggerFactory.getLogger(AplCoreRuntime.class);
     private List<AplCore> cores = new ArrayList<>();
```

Comment:
May be better to make it thread safe ? 
private List<AplCore> cores = Collections.synchronizedList( new ArrayList<>() );

Reply:
> May be better to make it thread safe ?
> private List cores = Collections.synchronizedList( new ArrayList<>() );

Though it is not a subject of this PR, I agree. Well, please do it in one of your branches or I'll do it later